samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json

```diff
@@ -0,0 +1,33 @@
+{
+   "properties": {
+     "displayName": "Audit SQL Database is Zone Redundant",     
```

Comment:
>Audit SQL Database is Zone Redundant [](start = 21, length = 36)

Do you mind changing the display name to "SQL databases should be zone redundant" to conform to newer naming standards?